server/plugin/command/view_calendar.go

```diff
@@ -15,8 +15,23 @@ func (c *Command) viewCalendar(parameters ...string) (string, error) {
 		return "", err
 	}
 
+	var timeZone string
+	tz, err := c.API.GetUserTimezone(c.Args.UserId)
+	if err == nil {
+		timeZone = tz
+	}
+
+	if timeZone != "" {
+		for _, event := range events {
+			event.Start = event.Start.ConvertToTimezone(timeZone)
```

Comment:
y, I think `event.Start.InTimezone(timeZone)`, or even better `event.Start.In(timeZone)` would be typographically better, and sufficiently intention-revealing. (see comments below, reading from the bottom)

Reply:
`ConvertToTimezone` is too Java, not poetic enough :)  (poetic == reduced, simplified)